apps/evm/src/hooks/useOperationModal/Modal/index.tsx

```diff
@@ -83,7 +83,11 @@ const OperationModal: React.FC<OperationModalProps> = ({
     <Modal
       isOpen
       title={
-        <TokenIconWithSymbol token={vToken.underlyingToken} className="text-lg font-semibold" />
+        <TokenIconWithSymbol
+          token={vToken.underlyingToken}
+          useNativeTokenAsset
```

Comment:
I see, thanks for the context.
I think displaying the same icon all the time makes more sense, otherwise we risk confusing users. Thanks for clarifying this with Jesús.